js/spa/test/test_chatinterface.reload.spec.ts

```diff
@@ -49,38 +48,39 @@ test("gradio dev mode correctly reloads a stateful ChatInterface demo", async ({
 	try {
 		const { _process: server_process, port: port } =
 			await launch_app_background(
-				`gradio ${join(process.cwd(), demo_file)}`,
+				`gradio ${join(process.cwd(), "run.py")}`,
 				process.cwd()
 			);
 		_process = server_process;
 		console.log("Connected to port", port);
 		const demo = `
-import gradio as gr
+		import gradio as gr
```

Comment:
I think we need to remove this indentation otherwise the python interpreter will freak out

```
  File "/Users/freddy/sources/gradio/gradio/utils.py", line 186, in _remove_no_reload_codeblocks
    tree = ast.parse(code)
           ^^^^^^^^^^^^^^^
  File "/Users/freddy/miniconda3/envs/gradio-311/lib/python3.11/ast.py", line 50, in parse
    return compile(source, filename, mode, flags,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<unknown>", line 2
    	import gradio as gr
    ^
IndentationError: unexpected indent
```